docs/micropython.md

```diff
@@ -11,9 +11,10 @@ MicroPython, we recommend you run your code with it for improved performance.
 Module                          Support status
 ==============================  =================
 `ev3dev2.button`                ️️✔️
+`ev3dev2.console`               ✔️️
 `ev3dev2.control` [1]_          ⚠️
-`ev3dev2.display`               ❌
-`ev3dev2.fonts` [2]_            ⚠️
+`ev3dev2.display` [2]_          ❌
```

Comment:
Shouldn't this be the yield sign for display since we partially support it?

Reply:
With this PR, where Console() is a separate class--it leaves Display() alone and thus still not supported in micropython (it depends on the FrameBuffer mmap goodness that micropython doesn't fully support apparently.)